lib/triton_kernel_call.h

```diff
@@ -0,0 +1,62 @@
+/* Copyright 2022 Google LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+#include <string>
+#include <pybind11/pybind11.h>
+#include <pybind11/stl.h>
+
+#include "cuda.h"
+
+typedef std::unordered_map<std::string, pybind11::object> asm_map_t;
+
+const int TRITON_MAX_N_SHARED_BYTES = 49152;
+const int TRITON_MAX_SHARED_OPTIN = 49152;
+
+class TritonExecutable {
```

Comment:
Wrap the code in a namespace. like namespace jax {...}

Reply:
Done.